### PR TITLE
feat(frontend): ConvertNetwork component

### PR DESCRIPTION
**Pull request by DenysKarmazynDFINITY**
# Motivation

`ConvertNetwork` component in action for both source and destination tokens:

Screenshot 2024-11-12 at 18 36 19

### CODE REVIEW

File: src/frontend/src/lib/components/convert/ConvertNetwork.svelte [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+	import { replacePlaceholders } from '$lib/utils/i18n.utils.js';
+
+	export let token: Token;
+</script>
+
+<ConvertValue>
+	<slot slot="label" name="label" />
+
+	<div class="flex" slot="main-value">
+		<Logo
```
Comment (AntonioVentilii) [inline src/frontend/src/lib/components/convert/ConvertNetwork.svelte:15]: not for this PR, but I am pretty sure we are using the same network BW logo elsewhere too, on more occasions. Is it worth to create a component for it?

**Review by AntonioVentilii — APPROVED** [review]
LGTM tks!

File: src/frontend/src/lib/components/convert/ConvertNetwork.svelte (hunk 1 above)
Comment (DenysKarmazynDFINITY) [reply src/frontend/src/lib/components/convert/ConvertNetwork.svelte:15]: Sounds good, the next time it's needed - let's create a common component for that

Comment (AntonioVentilii) [reply src/frontend/src/lib/components/convert/ConvertNetwork.svelte:15]: Done with:

- PR #3604

Comment (DenysKarmazynDFINITY) [reply src/frontend/src/lib/components/convert/ConvertNetwork.svelte:15]: Nice, thanks!

**Pull request merged**